lighthouse-core/test/results/sample_v2.json

```diff
@@ -3416,5 +3416,144 @@
       "title": "Crawling and Indexing",
       "description": "To appear in search results, crawlers need access to your app."
     }
+  },
+  "localeLog": {
+    "lighthouse-core/audits/metrics/interactive.js!#title": {
+      "occurrences": [
+        {
+          "path": [
+            "audits",
+            "interactive",
+            "title"
+          ]
+        }
+      ]
+    },
+    "lighthouse-core/audits/metrics/interactive.js!#description": {
+      "occurrences": [
+        {
+          "path": [
+            "audits",
+            "interactive",
+            "description"
+          ]
+        }
+      ]
+    },
+    "lighthouse-core/lib/i18n.js!#ms": {
+      "occurrences": [
+        {
+          "values": {
+            "timeInMs": 4927.278
+          },
+          "path": [
+            "audits",
+            "interactive",
+            "displayValue"
+          ]
+        }
+      ]
+    },
+    "lighthouse-core/audits/byte-efficiency/render-blocking-resources.js!#title": {
+      "occurrences": [
+        {
+          "path": [
+            "audits",
+            "render-blocking-resources",
+            "title"
+          ]
+        }
+      ]
+    },
+    "lighthouse-core/audits/byte-efficiency/render-blocking-resources.js!#description": {
+      "occurrences": [
+        {
+          "path": [
+            "audits",
+            "render-blocking-resources",
+            "description"
+          ]
+        }
+      ]
+    },
+    "lighthouse-core/audits/byte-efficiency/render-blocking-resources.js!#displayValue": {
+      "occurrences": [
+        {
+          "values": {
+            "timeInMs": 1129,
+            "itemCount": 5
+          },
+          "path": [
+            "audits",
+            "render-blocking-resources",
+            "displayValue"
+          ]
+        }
+      ]
+    },
+    "lighthouse-core/lib/i18n.js!#columnURL": {
```

Comment:
how about [JSONPath](http://goessner.net/articles/JsonPath/index.html#e2)?


```js
    "lighthouse-core/lib/i18n.js!#columnURL": {
      "occurrences": [
        {
          "path": [
            "audits",
            "render-blocking-resources",
            "details",
            "headings",
            "0",
            "label"
          ]
        }
      ]
    },
```

could become:

```js
    "lighthouse-core/lib/i18n.js!#columnURL": {
      "occurrences": ["$.audits['render-blocking-resources'].details.headings[0].label"],
    },
```

obviously it's more dense, but I think its a lot more readable. 

(actually looks like [lodash's get](https://lodash.com/docs/4.17.10#get) also supports this style, anyhow.) 
We don't really need ALL of jsonpath, just the basics.

Reply:
do you want JSONPath or [what lodash get supports](https://github.com/lodash/lodash/blob/4.17.5/lodash.js#L6727-L6736)? ;)

I'm on board for this 👍

Reply:
I should've said, "I'm on board for this, so long as we use someone else's getter code :)"

Reply:
sg! based on their docs, the string form with lodash get lgtm